src/Integration/UserContext87.php

```diff
@@ -0,0 +1,45 @@
+<?php
+declare(strict_types=1);
+namespace Helhum\SentryTypo3\Integration;
+
+use Sentry\Event;
+use TYPO3\CMS\Core\Authentication\AbstractUserAuthentication;
+use TYPO3\CMS\Core\Authentication\BackendUserAuthentication;
+use TYPO3\CMS\Core\Core\Environment;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication;
+
+class UserContext87 implements ContextInterface
+{
+    public function appliesToEvent(Event $event): bool
+    {
+        return class_exists(Environment::class) === false
+            && defined('TYPO3_REQUESTTYPE')
+            && defined('TYPO3_REQUESTTYPE_CLI')
+            && TYPO3_REQUESTTYPE !== TYPO3_REQUESTTYPE_CLI
```

Comment:
I think that's a wrong check here. Should be: `TYPO3_REQUESTTYPE & TYPO3_REQUESTTYPE_CLI`, as `TYPO3_REQUESTTYPE` is a bitmask

Reply:
This file was removed.